srcpkgs/wget2/template

```diff
@@ -0,0 +1,53 @@
+# Template file for 'wget2'
+pkgname=wget2
+version=2.0.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config texinfo pandoc"
+makedepends="gnutls-devel zlib-devel libidn2-devel libpsl-devel
+ $(vopt_if brotli 'brotli-devel') $(vopt_if http2 'nghttp2-devel')
+ $(vopt_if gpgme 'gpgme-devel') $(vopt_if lzma 'liblzma-devel')
+ $(vopt_if bzip2 'bzip2-devel') $(vopt_if zstd 'libzstd-devel')
+ $(vopt_if pcre 'pcre2-devel')"
+depends="libwget>=${version}_${revision}"
+checkdepends="libmicrohttpd-devel gpgme"
+short_desc="GNU file and recursive website downloader"
+maintainer="chili-b <dexter.gaonshatford@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/gnuwget/wget2"
+changelog="https://gitlab.com/gnuwget/wget2/-/blob/master/NEWS"
+distfiles="${GNU_SITE}/wget/wget2-${version}.tar.gz"
+checksum=0bb7fa03697bb5b8d05e1b5e15b863440826eb845874c4ffb5e32330f9845db1
+
+build_options="brotli http2 gpgme lzma bzip2 zstd pcre"
+build_options_default="brotli http2 gpgme lzma bzip2 zstd pcre"
+
+post_install() {
+	rm ${DESTDIR}/usr/bin/wget2_noinstall
+	install -D -m 644 -t ${DESTDIR}/usr/share/man/man3 docs/man/man3/*.3
+}
+
+libwget_package() {
+	depends="gnutls libidn2 libpsl
+	 $(vopt_if brotli 'brotli') $(vopt_if http2 'nghttp2')
+	 $(vopt_if gpgme 'libgpgme') $(vopt_if lzma 'liblzma')
+	 $(vopt_if bzip2 'bzip2') $(vopt_if zstd 'libzstd')
+	 $(vopt_if pcre 'libpcre2')"
```

Comment:
Keep in mind that xbps can automatically get some of the dependencies using `objdump -p`.

Reply:
which could be omitted?

Reply:
I see
```
  NEEDED               libidn2.so.0
  NEEDED               libzstd.so.1
  NEEDED               libbrotlidec.so.1
  NEEDED               libz.so.1
  NEEDED               libnghttp2.so.14
  NEEDED               libpsl.so.5
  NEEDED               libgnutls.so.30
  NEEDED               libpthread.so.0
  NEEDED               libc.so.6
```
in the output of objdump -p. I guess these can be removed from the list of dependencies?

Reply:
as for the libraries which aren't, do these need to stay in `depends`, or does their absence mean they were linked statically and can therefore be removed as well?

Reply:
You don't need to do objdump manually to check, when building the package it lists them in `running pre-pkg hook: 04-generate-runtime-deps ...`.